src/index.html

```diff
@@ -13,23 +13,18 @@ <h1>Leaderboard</h1>
   <main>
     <section class="list">
       <div class="re-set"><h4>Recent Scores</h4>
-        <button type="reset">Refresh</button>
+        <button type="reset" class="refresh-bttn">Refresh</button>
       </div>
       <div class="score-table">
-        <ul>
-          <li>Name: 100</li>
-          <li>Name: 20</li>
-          <li>Name: 50</li>
-          <li>Name: 78</li>
-          <li>Name: 70</li>
-        </ul>
+        <ul class="scoreList">
+       </ul>
       </div>
     </section>
     <section>
       <h4>Add Your Score</h4>
       <form action="">
-        <input type="text" placeholder="Your name" name="name">
-        <input type="text" placeholder="Your score" name="name">
+        <input type="text" placeholder="Your name" name="name" class="uName">
+        <input type="text" placeholder="Your score" name="name" class="uScore">
```

Comment:
### OPTIONAL
- [ ] It will be good if you change the score type to the number. Right now ı can add a text as a score.

Reply:
Thank you